internal/command/operator/apply.go

```diff
@@ -102,11 +104,13 @@ Note: If --auto-registry-credentials and --registry-credentials-path are unset,
 				return fmt.Errorf("error validating provided flags: %w", err)
 			}
 
-			// issuers, err := extractIssuersFromBackupFile(backupFilePath)
-			// if err != nil {
-			// 	return fmt.Errorf("error extracting issuers from backup file: %w", err)
-			// }
-			// fmt.Println(issuers)
+			issuers, err := restore.ExtractOperatorManageableIssuersFromBackupFile(backupFilePath)
+			if err != nil {
+				return fmt.Errorf("error extracting issuers from backup file: %w", err)
+			}
+			if len(issuers.MissedIssuers) != 0 {
+				fmt.Fprintf(os.Stderr, "The following issuers cannot be managed by the operator and must be restored manually: %s\n", strings.Join(issuers.MissedIssuers, ", "))
+			}
```

Comment:
Suggestion: I think we should not error out, it suggests that the generated backup is invalid which is not the case. I think we should just print out clear 'next steps' as we already do for one of the other commands.

Reply:
This is just a warning rather than an error here. Do you still want to handle this differently?

Reply:
yeah I think the fact that it will send to stderr will suggest that it's a halting error and if we just sent this to stdout it'd be more obvious that the command finished everything it could and here are the next steps user should do.. lmk if you think otherwise

Reply:
I have made this print to stdout in 889116d9353c89103ef86bd7626f42d4d47ac8d8